index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" media="screen and (min-width: 1024px)" href="./desktop.css" />
+    <title> My Portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="main">
+            <header>
+                <div class="main-logo"><a class="logo" href="./index.html"> John Doe</a>
+                </div>
+                <nav>
+                    <ul class="menu-bar">
+                        <li class="menu-items">Portfolio</li>
+                        <li class="menu-items">About</li>
+                        <li class="menu-items">Contact</li>
+                        <li><img src="./png/nav-icon.png" alt="Multi-Post Stories"></li>
+                    </ul>
+                    <div class="hamburger" onclick="displayContent()">
+                        <img src="./svg/menu.svg" alt="Menu" />
+                    </div>
+                </nav>
+            </header>
+            <section class="main-section" id="portfolio">
+                <div class="background-adjust">
+                    <div class="details">
+                        <h1>Hey There. I`m Esmatullah</h1>
+                        <h2>I`am a Software Developer</h2>
+                        <p>
+                            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don`t hestiate to contact me.
+                        </p>
+                    </div>
+
+                    <ul class="social-media">
+                        <li><img src="./svg/github.svg" alt="github" /></li>
+                        <li><img src="./svg/linkedin.svg" alt="linkedin" /></li>
+                        <li><img src="./svg/Vector3.svg" alt="angelist" /></li>
+                        <li><img src="./svg/twitter.svg" alt="twitter" /></li>
+                        <li><img src="./svg/Vector5.svg" alt="medium" /></li>
+                    </ul>
```

Comment:
- [ ] Please remove the left spacing given to the `ul` tag by default by the browser so that your design matches the Figma template.
![image](https://user-images.githubusercontent.com/62290545/170528443-e212db54-9265-494a-9ae4-e644f7e1dcc0.png)

- [ ] Kindly wrap these social images in `a` tags so that they become clickable. Your code should look something like the following: 
```
<ul class="social-media">
    <li>
       <a>
          <img />
       </a>
    </li>
    <li>
       <a>
          <img />
       </a>
    </li>
    ...
</ul>
```

Reply:
Done as advised!